extensions/ext-codemirror/package.json

```diff
@@ -0,0 +1,18 @@
+{
+    "name": "ext-codemirror",
+    "version": "1.0.0", 
+    "description": "CodeMirror code editor integration for SVG-Edit",
+    "main": "ext-codemirror.js",
+    "keywords": [
+        "svg-edit",
+        "extension",
+        "codemirror",
+        "editor"
+    ],
+    "author": "Hemanta Sapkota <hemanta@yolmo.com>",
+    "license": "MIT",
+    "dependencies": {
+        "codemirror": "^5.65.2"
+    },
```

Comment:
No need to include `codemirror` as a dependency because we have packaged a pre-compiled bundle `CodeEditor.bundle.js`